cli/src/main/java/com/containersol/minimesos/main/Main.java

```diff
@@ -67,69 +69,87 @@ public static void main(String[] args) {
         }
     }
 
-    public Main() {
-        jc = new JCommander(this);
-        jc.setProgramName("minimesos");
-    }
 
     public void setOutput(PrintStream output) {
         this.output = output;
     }
 
     public int run(String[] args) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "run" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)